doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc

```diff
@@ -7,7 +7,7 @@ You can create a report with the link:https://cloud.redhat.com/beta[Migration An
 
 .Prerequisites
 
-* Inventory data file generated by Red Hat CloudForms or a `tar.gz` archive containing multiple inventory data files
+*  A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms
```

Comment:
' A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms'

Change to '`tar.gz` archive file containing one or more `.json` inventory data files'

I think that is sufficient.

Reply:
Please squash your commits: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.epes4xf06pzk

Reply:
Done.